crypto/bio/bio.c

```diff
@@ -236,6 +256,28 @@ int BIO_write(BIO *bio, const void *in, int inl) {
   return ret;
 }
 
+int BIO_write_ex(BIO *bio, const void *data, size_t data_len, size_t *written_bytes) {
+  if (bio == NULL) {
```

Comment:
Missing `written_bytes` null check

Reply:
It's a bit misaligned, but unlike `BIO_read_ex`, `written_bytes` is allowed to be NULL in OpenSSL. I've also documented this in the function description.